python/ray/air/_internal/util.py

```diff
@@ -113,3 +114,8 @@ def run(self):
     def join(self, timeout=None):
         super(RunnerThread, self).join(timeout)
         return self._ret
+
+
+def _estimate_avail_object_store_memory() -> int:
+    """Estimates total object store memory available in the cluster."""
+    return ray.available_resources()["object_store_memory"]
```

Comment:
Hmm but I don't think we account object store memory in placement groups, right? Or what did you mean exactly?

Reply:
The util tries to calculate what fraction of a cluster should be "used" by the placement group by looking at CPU and GPU cores. We could use that same estimated fraction to downsize the object store memory usage proportionally.

It's not a great solution, but probably better than using the entire cluster memory size for each trial when running Ray tune.

Reply:
Ah I see. Hmm I think this is a bit different because we're requiring the user to set a max memory fraction; this util is just estimating the static total object store capacity. I was imagining that AIR would just divide the user-specified fraction among the trials if Tune is being used (can do this in a follow-up PR), which seems simpler than trying to estimate on the fly.

Reply:
Yeah, we can revisit this. For the streaming executor, we'll probably have separate autodetection logic anyways.